lib/mods/theme/edit/ab_info.txt

```diff
@@ -86,13 +86,6 @@ A:102:Far reaching attack
 k:15:Combat
 k:15:Polearm-mastery
 
-N:9:Trapping
```

Comment:
So this robs rogue-type characters of a skill... can we think of anything to give them to compensate?